### PR TITLE
Create Dockerfiles for OBAM, OBKM and OBBI for Ubuntu and Alpine for release 2.0.0

### LINKED ISSUE #90: Create Docker resources for WSO2 Open Banking version 2.0.0
**Description:**
Create Dockerfiles for OBAM, OBKM and OBBI for Ubuntu based on adoptopenjdk:8u262-b10-jdk-hotspot-bionic image and for Alpine based on adoptopenjdk/openjdk8:jdk8u272-b10-alpine image. This effort is towards the OB 2.0 docker release.

**Affected Product Version:**
OB 2.0.0

**OS, DB, other environment details and versions:**
Ubuntu, Alpine

### PR DESCRIPTION
**Pull request by nalakagunathilaka**
## Purpose
Create Dockerfiles for OBAM, OBKM and OBBI for Ubuntu based on adoptopenjdk:8u272-b10-jdk-hotspot-focal image and for Alpine based on adoptopenjdk/openjdk8:jdk8u272-b10-alpine image. This fixes #90 

## Goals
Create Dockerfiles for OBAM, OBKM and OBBI to start the servers with default configurations

## Test environment
Ubuntu, Alpine with mentioned JDKs above.

### CODE REVIEW

File: dockerfiles/alpine/obam/Dockerfile [hunk 1]
```diff
@@ -64,6 +64,7 @@ RUN \
     apk add --no-cache \
         bash \
         libxml2-utils \
+        unzip \
```
Comment (chirangaalwis) [inline dockerfiles/alpine/obam/Dockerfile:67]: @nalakagunathilaka this should come to the bottom. The Dockerfile best practice is to maintain the package list in the alphabetical order of the starting letter.

File: dockerfiles/alpine/obam/Dockerfile [hunk 2]
```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk8:jdk8u242-b08-alpine
+FROM adoptopenjdk/openjdk8:jdk8u272-b10-alpine
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-open-banking/releases/tag/v1.5.0.2"
+      com.wso2.docker.source="https://github.com/wso2/docker-open-banking/releases/tag/v2.0.0"
```
Comment (chirangaalwis) [inline dockerfiles/alpine/obam/Dockerfile:22]: @nalakagunathilaka this version will be `v2.0.0.1`. Please apply the change to all files.

File: dockerfiles/ubuntu/obbi/worker/Dockerfile [hunk 3]
```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Ubuntu Docker image
-FROM adoptopenjdk:8u242-b08-jdk-hotspot-bionic
+FROM adoptopenjdk:8u262-b10-jdk-hotspot-bionic
```
Comment (chirangaalwis) [inline dockerfiles/ubuntu/obbi/worker/Dockerfile:20]: @nalakagunathilaka the update number of AdoptOpenJDK has to be uniform -> please use the [focal image](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8u272-b10-jdk-hotspot-focal/images/sha256-ec12ef973c20bda37ef09fb63477e62fe3d0784a3072609c91162c3e0d46ec39?context=explore).

**Review by chirangaalwis — CHANGES_REQUESTED** [review]

File: dockerfiles/alpine/obam/Dockerfile (hunk 1 above)
Comment (nalakagunathilaka) [reply dockerfiles/alpine/obam/Dockerfile]: Resolved with latest commit.

File: dockerfiles/alpine/obam/Dockerfile (hunk 2 above)
Comment (nalakagunathilaka) [reply dockerfiles/alpine/obam/Dockerfile]: Resolved with latest commit.

File: dockerfiles/ubuntu/obbi/worker/Dockerfile (hunk 3 above)
Comment (nalakagunathilaka) [reply dockerfiles/ubuntu/obbi/worker/Dockerfile]: Resolved with latest commit.

**Review by chirangaalwis — APPROVED** [review]

**Review by chirangaalwis — APPROVED** [review]

**Pull request merged**